src/private/WRLDEntityHighlightsApiHelpers.mm

```diff
@@ -0,0 +1,37 @@
+#include "WRLDEntityHighlightsApiHelpers.h"
```

Comment:
possibly rename WRLDIndoorEntityHelpers? Private impl though, may not matter.
These actually seem like more general helpers, just transforming to and from NSArray of NSStrings to std::vector<std::string> - could it be moved + methods renamed to be more general?